pyproject.toml

```diff
@@ -111,6 +111,12 @@ exclude_lines = [
   "@typing.overload",
   "@overload",
 
+  # Protocols
+  "@runtime_checkable",
```

Comment:
this code is never executed directly, only introspection is used to match target objects